tests/gold_tests/headers/cachedIMSRange.test.py

```diff
@@ -64,7 +64,6 @@
     'proxy.config.diags.debug.tags': 'http',
     'proxy.config.http.response_via_str': 3,
     'proxy.config.http.cache.http': 1,
-    'proxy.config.http.wait_for_cache': 1,
 })
```

Comment:
What is the logic for removing wait_for_cache here?

Reply:
Thanks for the review!

I pushed wait_for_cache into MakeATSProcess. The logic being if you want the cache enabled, then you'll want to wait for it to be enabled before using ATS.

See:
https://github.com/apache/trafficserver/pull/6931/files#diff-bdc07bfafa700b29a03a4b96413198acR294

Reply:
Looks good.